app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainFragment.kt

```diff
@@ -36,13 +36,7 @@ class MainFragment : Fragment() {
         super.onActivityCreated(savedInstanceState)
         viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
         viewModel.stocks.observe(viewLifecycleOwner, Observer { stocks ->
-            actStock.setAdapter(
-                ArrayAdapter(
-                    context!!,
-                    R.layout.support_simple_spinner_dropdown_item,
-                    stocks
-                )
-            )
+            actStock.setAdapter(ArrayAdapter(context!!, R.layout.support_simple_spinner_dropdown_item, stocks))
```

Comment:
To make sure the code looks "less" as possible I put the lambda expression in one line. Even though the code before used good trailing commas, lambdas can help make the space around code be smaller, as discussed in the lecture videos and this article https://kotlinlang.org/docs/coding-conventions.html#lambdas

Reply:
Either way works.